gcc/rust/operator.h

```diff
@@ -1,69 +1,54 @@
-// operator.h -- Go frontend operators.     -*- C++ -*-
+#ifndef OPERATOR_H
+#define OPERATOR_H
 
-// Copyright 2009 The Go Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style
-// license that can be found in the LICENSE file.
-
-#ifndef GO_OPERATOR_H
-#define GO_OPERATOR_H
-
-// The operators.
+enum class NegationOperator
+{
+  NEGATE,
+  NOT
+};
 
-// TODO: Will have to be significantly modified to work with Rust and current
-// setup of gccrs
+enum class ArithmeticOrLogicalOperator
+{
+  ADD,	       // std::ops::Add
```

Comment:
is it just github or is the formatting off for the comment?

Reply:
I think it's a GitHub problem. The formatting is controlled by Clang-format, and they look fine on my editor.

The whitespace presented here is one tab followed by 7 space. Other lines that contain tabs also look off on GitHub, even before the pull request.